packages/database/src/migrations/20211001060307-LESMISDropOutRateProvinceOverlay-modifies-data.js

```diff
@@ -0,0 +1,311 @@
+'use strict';
+
+import { insertObject, generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const PERMISSION_GROUP = 'LESMIS Public';
+const PROVINCE_OVERLAY_GROUP_ID = '5f2c7ddc61f76a513a0001f3';
+
+const getOverlayNames = (gradeNum, educationLevel, GPI) => {
+  if (gradeNum !== null) {
+    return {
+      name: `Grade ${educationLevel === 'p' ? gradeNum : gradeNum + 5} Dropout Rate ${
+        GPI ? 'GPI' : ''
+      }`,
+      dataElement: [
+        `dor_province_${educationLevel.concat(gradeNum)}_${GPI ? 'f' : 't'}`,
+        `${GPI ? 'dor_province_s1_m' : ''}`,
```

Comment:
```suggestion
        `${GPI ? 'dor_province_s1_m' : ''}`,
        ... GPI ? ['dor_province_s1_m'] : [],
```

The way this is written currently these reports will wind up with an empty string in their dataElements array, I'm not sure if that's actually a problem but it's a little messy

Reply:
Thanks for the tip, will update that.